runtime/src/accounts_background_service.rs

```diff
@@ -629,26 +629,38 @@ impl AccountsBackgroundService {
                                     break;
                                 }
                             }
-                        } else if previous_clean_time.elapsed() > CLEAN_INTERVAL {
-                            // Note that the flush will do an internal clean of the
-                            // cache up to bank.slot(), so should be safe as long
-                            // as any later snapshots that are taken are of
-                            // slots >= bank.slot()
-                            bank.force_flush_accounts_cache();
```

Comment:
this uses `bank.slot()`, not `bank.slot()-1`:
https://github.com/anza-xyz/agave/blob/347b25d7bf159a7074b9d9f0db83c58cacdd6853/runtime/src/bank.rs#L4828

Reply:
may not matter, but is different.

Reply:
ok, so you want the values to match. I imagine this is 'fine' to flush 1 less slot potentially.
It looks like it is 'inclusive':
https://github.com/anza-xyz/agave/blob/347b25d7bf159a7074b9d9f0db83c58cacdd6853/accounts-db/src/accounts_db.rs#L5953